src/pocketdb/web/PocketContentRpc.cpp

```diff
@@ -1596,4 +1596,64 @@ namespace PocketWeb::PocketWebRpc
     },
         };
     }
+
+    RPCHelpMan GetsubsciptionsGroupedByAuthors()
+    {
+        return RPCHelpMan{"GetsubsciptionsGroupedByAuthors",
+                          "\n\n", // TODO (rpc)
+                          {
+                                  {"address", RPCArg::Type::STR, RPCArg::Optional::NO, "" /* TODO (rpc): arg description*/},
+                                  {"addressPagination", RPCArg::Type::STR, RPCArg::Optional::OMITTED_NAMED_ARG, "" /* TODO (rpc): arg description*/},
+                                  {"nHeight", RPCArg::Type::NUM, RPCArg::Optional::OMITTED_NAMED_ARG, "" /*TODO (rpc): arg description*/},
+                                  {"countOutOfUsers", RPCArg::Type::NUM, RPCArg::Optional::OMITTED_NAMED_ARG, "" /* TODO (rpc): arg description*/},
+                                  {"countOutOfcontents", RPCArg::Type::STR, RPCArg::Optional::OMITTED_NAMED_ARG, "" /* TODO (rpc): arg description*/}
+                          },
+                          {
+                                  // TODO (rpc): provide return description
+                          },
+                          RPCExamples{
+                                  // TODO (rpc): better examples
+                                  HelpExampleCli("getsubsciptionsgroupedbyauthors", "...") +
+                                  HelpExampleRpc("getsubsciptionsgroupedbyauthors", "...")
+                          },
+                          [&](const RPCHelpMan& self, const JSONRPCRequest& request) -> UniValue
+                          {
+                              string address = "";
+                              string addressPagination = "";
+                              int nHeight = ChainActive().Height();
+                              int countOutOfUsers = 10;
+                              int countOutOfcontents = 10;
+
+                              UniValue result(UniValue::VOBJ);
+
+                              if (request.params.empty())
+                                  return result;
+
+                              if (request.params[0].isStr())
```

Comment:
I'm not a c++ -ninja yet, but I think this part until line 1645 may be rewritten more elegant way somehow :)